specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1174 @@
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a private cloud resource",
+          "$ref": "#/definitions/PrivateCloudProperties"
+        }
+      }
+    },
+    "PrivateCloudProperties": {
+      "description": "The properties of a private cloud resource",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Pending",
+            "Building",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "PrivateCloudProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "circuit": {
+          "description": "An ExpressRoute Circuit",
+          "$ref": "#/definitions/Circuit"
+        },
+        "cluster": {
+          "description": "The default cluster used for management",
+          "$ref": "#/definitions/DefaultClusterProperties"
+        },
+        "clusters": {
+          "description": "The clusters",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        },
+        "endpoints": {
+          "description": "The endpoints",
+          "readOnly": true,
+          "$ref": "#/definitions/Endpoints"
+        },
+        "internet": {
+          "description": "Connectivity to internet is enabled or disabled",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "InternetEnum",
+            "modelAsString": true
+          }
+        },
+        "identitySources": {
+          "description": "vCenter Single Sign On Identity Sources",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IdentitySource"
+          }
+        },
+        "networkBlock": {
+          "description": "The block of addresses should be unique across VNet in your subscription as well as on-premise. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22",
+          "type": "string"
+        },
+        "managementNetwork": {
+          "description": "Network used to access vCenter Server and NSX-T Manager",
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningNetwork": {
+          "description": "Used for virtual machine cold migration, cloning, and snapshot migration",
+          "type": "string",
+          "readOnly": true
+        },
+        "vmotionNetwork": {
+          "description": "Used for live migration of virtual machines",
+          "type": "string",
+          "readOnly": true
+        },
+        "vcenterPassword": {
```

Comment:
>vcenterPassword [](start = 9, length = 15)

Are you returning passwords on GETs?

Reply:
No. Passwords may be listed using `PrivateClouds_ListAdminCredentials`.

Reply:
Excellent. Thanks for confirming.